Reference/Templating/Modelsbuilder/index.md

```diff
@@ -7,7 +7,8 @@ A tool that can generate a complete set of strongly-typed published content mode
 * [Builder Modes](Builder-Modes.md)
 * [Understand And Extend Models](Understand-And-Extend.md)
 * [Control Models Generation](Control-Generation.md)
-* [IPublishedContentModelFactory](IPublishedContentModelFactory.md)  
-* [Cool things to do with models](CoolThingsWithModels.md)  
+* [IPublishedContentModelFactory](IPublishedContentModelFactory.md)
+* [Cool things to do with models](CoolThingsWithModels.md)
+* [Using Interfaces](Using-Interfaces.md)
```

Comment:
I don't want to be too detail oriented, but I feel that 'Cool things to do with models' is very unspecific and as such should be the bottom one under all the specific things 😄 Let me know what you think.

Reply:
Me too! That’s also how I thought I did it :-)
In my head, as I can clearly see :D

I’ll swap them around ASAP